VERSION

```diff
@@ -0,0 +1 @@
+1.0.0-pre1
```

Comment:
@rmarianski Is it possible to do the same for the tilejson file template, or are we considering that the SERVICE version? Where should the service version go?